help/md/fnGTR.md

```diff
@@ -1,20 +1,40 @@
 ## name
 fnGTR
+
 ## title
 The General Time-Reversible rate matrix
+
 ## description
-The GTR rate matrix.
+DNA evolution model proposed in Tavare (1986).
+
 ## details
-The general time-reversible rate matrix:
-  Q(i,j) = exchangeRates(i,j) * baseFrequencies[j]
-The exchangeRates are symmetric.
+In this model, states are allowed to have different stationary frequencies, and exchangeability rates between states are allowed to be different. Its first argument, exchangeRates, codes for the transition rates between states (as in other models, transition rates are assumed to be symmetric). Its second argument, baseFrequencies, codes for the stationary frequencies of these states. Note that for n states, exchangeRates should have length n*(n-1)/2, and baseFrequencies should have length n. While this is usually used for DNA (and therefore has four states), the function can take any number of states, and therefore be used for many other applications (such as aminoacid or morphological evolution).
+
+The general time-reversible rate matrix elements will be of the form:
+     Q[i, j] = c * exchangeRates[i, j] * baseFrequencies[j]
+
+where c is a constant needed to normalize the average rate to 1.
+
 ## authors
 ## see_also
+fnJC
```

Comment:
Can you change this to be on the same line, like 
```
    fnJC, fnK80, fnK81, fnT92, fnHKY, fnTrN
```

This avoids wasting space, and also R tends to do this.